integration/pvtdata/pvtdata_test.go

```diff
@@ -779,203 +741,150 @@ var _ bool = Describe("PrivateData", func() {
-					expectedErrMsg := "tx creator does not have write access permission"
-					invokeChaincodeExpectErr(network, peer1, command, expectedErrMsg)
+				By("attempting to invoke chaincode from a user (org1) not in any collection member orgs (org2 and org3)")
+				peer2 := network.Peer("Org2", "peer0")
+				marbleDetailsBase64 := base64.StdEncoding.EncodeToString([]byte(`{"name":"memberonly-marble", "color":"blue", "size":35, "owner":"tom", "price":99}`))
+				command := commands.ChaincodeInvoke{
+					ChannelID:     channelID,
+					Orderer:       network.OrdererAddress(orderer, nwo.ListenPort),
+					Name:          "marblesp",
+					Ctor:          fmt.Sprintf(`{"Args":["initMarble"]}`),
+					Transient:     fmt.Sprintf(`{"marble":"%s"}`, marbleDetailsBase64),
+					PeerAddresses: []string{network.PeerAddress(peer2, nwo.ListenPort)},
+					WaitForEvent:  true,
+				}
+				peer1 := network.Peer("Org1", "peer0")
+				expectedErrMsg := "tx creator does not have write access permission"
+				invokeChaincodeExpectErr(network, peer1, command, expectedErrMsg)
 
-					assertMarbleAPIs()
-					assertDeliverWithPrivateDataACLBehavior()
-				})
+				assertMarbleAPIs()
+				assertDeliverWithPrivateDataACLBehavior()
 			})
 
-			When("chaincode in new lifecycle", func() {
-				It("calls marbles APIs and delivers private data", func() {
-					By("deploying new lifecycle chaincode")
-					testChaincode = chaincode{
-						Chaincode: newLifecycleChaincode,
-						isLegacy:  false,
-					}
-					nwo.EnableCapabilities(network, channelID, "Application", "V2_0", orderer, network.Peers...)
-					testChaincode.CollectionsConfig = collectionConfig("collections_config3.json")
-					deployChaincode(network, orderer, testChaincode)
-
-					By("attempting to invoke chaincode from a user (org1) not in any collection member orgs (org2 and org3)")
-					peer2 := network.Peer("Org2", "peer0")
-					marbleDetailsBase64 := base64.StdEncoding.EncodeToString([]byte(`{"name":"memberonly-marble", "color":"blue", "size":35, "owner":"tom", "price":99}`))
-					command := commands.ChaincodeInvoke{
-						ChannelID:     channelID,
-						Orderer:       network.OrdererAddress(orderer, nwo.ListenPort),
-						Name:          "marblesp",
-						Ctor:          fmt.Sprintf(`{"Args":["initMarble"]}`),
-						Transient:     fmt.Sprintf(`{"marble":"%s"}`, marbleDetailsBase64),
-						PeerAddresses: []string{network.PeerAddress(peer2, nwo.ListenPort)},
-						WaitForEvent:  true,
-					}
-					peer1 := network.Peer("Org1", "peer0")
-					expectedErrMsg := "tx creator does not have write access permission"
-					invokeChaincodeExpectErr(network, peer1, command, expectedErrMsg)
-
-					assertMarbleAPIs()
-					assertDeliverWithPrivateDataACLBehavior()
-				})
-			})
 		})
 
 		Describe("Collection Config Endorsement Policy", func() {
-			When("using legacy lifecycle chaincode", func() {
-				It("ignores the collection config endorsement policy and successfully invokes the chaincode", func() {
```

Comment:
This one actually tests a functional difference between old and new lifecycle.
I think only the tests that are true duplicates across old and new lifecycle (like the other tests above) should be removed.